R/derive_var_extreme_date.R

```diff
@@ -310,7 +310,7 @@ derive_var_extreme_dtm <- function(dataset,
     add_data[[i]] <- transmute(
```

Comment:
@pharmaverse/admiral Wondering if we should rewrite the code-section as well to keep up with our current programming practices but otherwise I think it should be fine?

Reply:
Hey @bms63 @manciniedoardo @bundfussr Are you able to provide some feedback on @zdz2101 question above, or should I merge this PR for now and the question above could be address in a separate issue?

Reply:
@zdz2101 What do you mean by code-section?  I'm not following :(

Reply:
@bms63 little two-parter, semi-related to function re-work type of stuff:
1) usually with the `set_values_to` argument related stuff, we use `process_set_values_to()` from `admiraldev` 
2) `dplyr::transmute()` was superseded earlier this year, we don't use it too often in our codebase anyhow but if we want to harmonize our code-base such that all functions are semi-consistently this is kinda part of it

Personal opinion, I'm okay with just merging this in, just thought I'd throw it out there. It'd probably be a back of the backlog type of task, kinda that mantra of don't fix wha'ts not broken type of thing

Reply:
My view is that harmonisation of the code base is important; within the harmonisation efforts priority should be given to user-facing harmonisation so that we can move faster towards admiral 1.0.0. Thus I don't mind if a change like this is relegated to the backlog and we get to it when we get to it.